snmp/datadog_checks/snmp/data/default_profiles/tripplite-ups.yaml

```diff
@@ -169,23 +171,24 @@ metrics:
         tag: tl_ups_outlet_group_desc
       - column:
           OID: 1.3.6.1.4.1.850.100.1.11.2.1.5
-          name: upsOutletGroupStatusGroupState
-        tag: ups_outlet_group_status_group_state
+          name: tlUpsOutletGroupState
+        tag: tl_ups_outlet_group_state
         mapping:
-          1: ups_outlet_group_status_on
-          2: ups_outlet_group_status_off
-          3: ups_outlet_group_status_unknown
-
-metric_tags:
-  - tag: tl_ups_ident_serial_num
-    OID: 1.3.6.1.4.1.850.100.1.1.2.0
-    symbol: tlUpsIdentSerialNum
-  - OID: 1.3.6.1.4.1.850.100.1.1.3.0
-    symbol: tlUpsIdentID
-    tag: tl_ups_ident_id
-  - OID: 1.3.6.1.4.1.850.100.1.1.4.0
-    symbol: tlUpsSnmpCardSerialNum
-    tag: tl_ups_snmp_card_serial_num
-  - OID: 1.3.6.1.4.1.850.100.1.1.6.0
-    symbol: tlUpsLocation
-    tag: tl_ups_location
+          0: unknown
+          1: off
+          2: on
+          3: mixed
+# Tagging by column OID is not supported at the moment. Should be a table/columns metric + tags instead.
```

Comment:
I'm seeing all the below tags as scalars in the MIB so I think we can keep them

Reply:
Ah small mix up, these one should be removed :
https://github.com/DataDog/integrations-core/pull/15668/files#diff-9c6aaa73fc0c919b02f03061746b54ded3e45c3e7ac2bf137b38ef63d2f1d68dR227-R236

Reply:
fixed !
thanks for the catch